eq-author/src/App/QuestionnaireDesignPage/index.test.js

```diff
@@ -236,14 +236,27 @@ describe("QuestionnaireDesignPage", () => {
       });
     });
 
-    it("should disable adding question confirmation when not on a question page", () => {
+    it("should disable adding question confirmation when not on a page", () => {
       match.params.entityName = "foo";
       wrapper.setProps({ match });
       expect(wrapper.find(NavigationSidebar).props()).toMatchObject({
         canAddQuestionConfirmation: false,
       });
     });
 
+    it("should disable adding question confirmation when not on a question page", () => {
+      questionnaire.sections[0].pages[0] = {
+        id: "1",
+        title: "",
+        pageType: "NotQuestionPage",
+        position: 0,
+      };
+      wrapper.setProps({ data: { questionnaire } });
+      expect(wrapper.find(NavigationSidebar).props()).toMatchObject({
```

Comment:
Can this be changed to test for specific props. [PR 326](https://github.com/ONSdigital/eq-author-app/pull/326/files#) adds other properties which would break this test. See file change [https://github.com/ONSdigital/eq-author-app/pull/326/files#diff-49e7391792e178cd6643a4c5073cc283](https://github.com/ONSdigital/eq-author-app/pull/326/files#diff-49e7391792e178cd6643a4c5073cc283) for reference